CHANGES.rst

```diff
@@ -1,4 +1,4 @@
-3.0 (unreleased)
+﻿3.0 (unreleased)
```

Comment:
I believe it's 3.0, we always release the first one in a series without micro version number.

However I wonder why this line has changed at all, please make sure you don't add unnecessary whitespaces.